lms/templates/problem_notifications.html

```diff
@@ -3,8 +3,8 @@
 <%! from django.utils.translation import ugettext as _ %>
 
 <div class="notification ${notification_type} ${'notification-'}${notification_name}"
-     tabindex="-1">
-    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
+     tabindex="-1" ${'style="display: none"'if notification_name == 'error' else '' } >
```

Comment:
I'm not accustomed to seeing style being set directly in the HTML. I believe the better practice is to use a class (is-hidden is common) and CSS.

@alisan617, please correct me if I am wrong here.